Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/svcalling/ShivaSvCallingSuccess.scala

```diff
@@ -0,0 +1,7 @@
+package nl.lumc.sasc.biopet.test.shiva.svcalling
+
+import nl.lumc.sasc.biopet.test.PipelineSuccess
+
+trait ShivaSvCallingSuccess extends ShivaSvCalling with PipelineSuccess {
```

Comment:
This pipeline also has a summary but leave this for now. I will change PipelineSuccess to SummaryPipeline in the new summary branch later. Maybe just add a todo in the code that we don't forget this.